subprojects/docs/src/snippets/testing/test-suite-plugin/kotlin/build.gradle.kts

```diff
@@ -35,7 +35,7 @@ testing {
             useJUnitJupiter() // <3>
         }
 
-        val integrationTest by registering(JvmTestSuite::class) { // <4>
+        register<JvmTestSuite>("integrationTest") { // <4>
```

Comment:
Why did this change?

Reply:
This was unrelated to these changes. It was failing in `master` but was marked as ignored in CI. The issue was that Kotlin was complaining about the `integrationTest` variable being unused.